docs/book/src/tasks/certs/auto-rotate-certificates-in-kcp.md

```diff
@@ -0,0 +1,64 @@
+## Automatically rotating certificates using Kubeadm Control Plane provider
+
+When using Kubeadm Control Plane provider (KCP) it is possible to configure automatic certificate rotations. KCP does this by triggering a rollout when the certificates on the machines are about to expire.
+
+Rollout for certificate expiry is automatically enabled for all new clusters created using ClusterAPI v1.3 and above. To enable it for existing clusters see [Enabling Certificate Expiry Rollout for Existing Clusters](#enabling-certificate-expiry-rollout-for-existing-clusters)
```

Comment:
>It happens on a per-machine basis, correct? Otherwise folks might think it's only on new clusters
Yes, it is on a machine basis.

>Not sure if we should say automatically enabled given that it's only enabled if certificatesExpiryDays is set
I re-worede the statement a little to adjust for this. WDYT?

Reply:
Sounds good!